build/azure-pipelines/linux/product-build-linux.yml

```diff
@@ -29,6 +29,11 @@ jobs:
       NPM_ARCH: ${{ parameters.NPM_ARCH }}
       VSCODE_ARCH: ${{ parameters.VSCODE_ARCH }}
     templateContext:
+      sdl:
+        binskim:
+          scanOutputDirectoryOnly: true
+          analyzeTargetGlob: '**/.build/**/*.node'
```

Comment:
are these two parameters `scanOutputDirectoryOnly` and `analyzeTargetGlob` exclusive ? Because `.build` directory will not be present in the out folder, it is usually under the src folder unless this is talking about the `.build` folder being published to the pipeline artifact. In either case it will not contain any `.node` files. Why not focus on the `.node` modules in the out folder instead specifically `$(Build.ArtifactStagingDirectory)/out/client` and `$(Build.ArtifactStagingDirectory)/out/server`, that is what we end up shipping ?